kuma/users/newsletter/checks.py

```diff
@@ -9,33 +9,43 @@
 SENDINBLUE_API_ERROR = "kuma.users.sendinblue.E001"
 
 
-def create_sendinblue_attributes():
-    response = sendinblue.request("GET", "contacts/attributes")
-    if not response.ok:
-        return [
-            Error(
-                f"Error getting sendinblue attributes: {response.status_code}",
-                id=SENDINBLUE_API_ERROR,
-            )
-        ]
+def create_missing_attributes(existing_attributes, required_attributes):
+    for name, sendinblue_type in required_attributes.items():
+        if any(attribute["name"] == name for attribute in existing_attributes):
+            continue
 
-    if not any(
-        attribute["name"] == "IS_PAYING" for attribute in response.json()["attributes"]
-    ):
         response = sendinblue.request(
-            "POST", "contacts/attributes/normal/IS_PAYING", json={"type": "boolean"},
+            "POST",
+            f"contacts/attributes/normal/{name}",
+            json={"type": sendinblue_type},
         )
         if not response.ok:
+            message = response.json()["message"]
```

Comment:
This scares me a little. If you get a 504 Bad Gateway, or whatever it's called, you might get a Nginx default error message. So it might not be JSON. 

I don't think we can rely on `response.json()` working. They might be a good in that their web server uses `application/json` on a 400 Bad Request maybe but `not response.ok` is also all sorts of other problems, no?

Reply:
On a scale from miserable to enthused, how do you feel about merging it in as is and see what sentry throws at us in the next months? I'm honestly curious about the prevalence of a non-JSON response here (which does sound plausible, no deniying that)